password.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Name of the Magento container
+CONTAINER_NAME="magento-magento-1"
```

Comment:
This also assumes that the project lives inside a folder called `magento`.

Reply:
This is also not the directory name. This is the name of the container which is set prior and will remain same everytime any one setup the code